src/patternfly/components/NotificationDrawer/notification-drawer-basic-task-list-hover-action-progress.hbs

```diff
@@ -0,0 +1,67 @@
+{{#> notification-drawer-task-list}}
+
+  {{#> notification-drawer-task-list-item notification-drawer-task-list-item--IsHoverable="true"}}
+    {{#> notification-drawer-task-list-item-title}}
+      hover action with progress bar status
+    {{/notification-drawer-task-list-item-title}}
+    {{#> notification-drawer-task-list-item-hover-action}}
+      Cancel
+    {{/notification-drawer-task-list-item-hover-action}}
+    {{#> notification-drawer-task-list-item-status}}
+      {{#> progress
```

Comment:
This is creating extra space to the right because the single line variation has a grid-gap, expecting the value to be on the right. I wonder if we need a variation for a progress without a value, or if this should include a value? wdyt @mcarrano?